gems/sorbet-runtime/test/types/configuration.rb

```diff
@@ -59,7 +60,7 @@ def self.receive(*); end
     describe 'sig_builder_error_handler' do
       describe 'when in default state' do
         it 'raises an error' do
-          @mod.sig {returns(Symbol).void}
+          @mod.sig {generated.returns(Symbol).checked(:always)}
```

Comment:
@ptarjan every change in this file is a revert of the change to disable generated / checked for open source.

Reply:
Yeah, I'm having a bunch of trouble keeping track of which side is the source of truth for any given change. I'll revert most of this file, but we have to stop this lunacy

Reply:
strong +1